From bacce9e2bfff7eddd11c5f37baf9b5fdf58a4250 Mon Sep 17 00:00:00 2001
From: Vadim Rutkovsky <vrutkovs@redhat.com>
Date: Wed, 25 Mar 2015 13:19:46 +0100
Subject: [PATCH 1/3] Don't check for xdg-email

xdg-email is required just once in the app and requires xdg-utils,
which fails to build with builddir!=srcdir
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0e224f3..cf72baa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -65,8 +65,8 @@ AC_MSG_RESULT($california_ok)
 AM_CONDITIONAL(HAVE__NL_TIME_FIRST_WEEKDAY, test "$california_ok" = "yes")
 
 # xdg-utils (specifically, xdg-email)
-AC_CHECK_PROG([XDG_EMAIL], [xdg-email], [yes], [no])
-AS_IF([test "x$XDG_EMAIL" != xyes], [AC_MSG_ERROR([xdg-email required. Please install xdg-utils package.])])
+#AC_CHECK_PROG([XDG_EMAIL], [xdg-email], [yes], [no])
+#AS_IF([test "x$XDG_EMAIL" != xyes], [AC_MSG_ERROR([xdg-email required. Please install xdg-utils package.])])
 
 #
 # configure switches
-- 
1.8.3.1

